recipes/pestpp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pestpp" %}
+{% set version = "5.2.6" %}
+{% set sha256 = "f1642df05fb7ee3e3a7601dfdcb68b74d0f3bb6eb4657a0b3e2b788e29dc63ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/pestpp/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.9
+    - make
+    - ninja
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
Why is Python a requirement? I see that it's used upstream in the CI for benchmarking, but does the library actually need it?

Reply:
I think it was left over from a template.  I removed both of them.